tools/pfe-tools/package.json

```diff
@@ -107,6 +107,7 @@
     "esbuild-plugin-lit-css": "^2.0.0",
     "esbuild-plugin-minify-html-literals": "^1.0.1",
     "eslint": "^8.14.0",
+    "execa": "^6.1.0",
```

Comment:
I don't see this being used anywhere in this PR.  Is this a leftover from another change either not included or removed?

Reply:
it's necessary to avoid an error in RHDS, which has it's own version of execa which differs by at least one semver major